src/siteconfig/tests/test_models.py

```diff
@@ -103,3 +107,15 @@ def test_SiteConfig_cache_expires(self):
         cache_time_expiration = localtime() + timedelta(days=1)
         with freeze_time(cache_time_expiration, tz_offset=0):
             self.assertIsNone(cache.get(SiteConfig.cache_key()))
+
+    def test_get_deck_owner(self):
```

Comment:
get_deck_owner() method can return three different values from 3 different logic branches.  Make sure each of those logic branches are tested and return the expected value.  Can do all in same test if simple, or three separate tests if more complicated.

Also, by the name of this test method I'm expecting it to be tests for that method (call that method with certain parameters and ensure the expected return value) but the method isn't directly called in this test.

This is testing the SiteConfig model itself and the user_owner field.  Should be called something like `test_siteconfig_deck_owner__default_value`

Reply:
done

Reply:
done